recipes/17/17_statement_sets.md

```diff
@@ -0,0 +1,104 @@
+# 17 Statement Sets 
+
+:bulb: In this recipe, you will see how to us [Statement Sets](https://docs.ververica.com/user_guide/sql_development/sql_scripts.html#sql-statements) run multiple queries in a single holistically optimized Flink Job. 
+
+Many product requirements involve outputting the results of a streaming application to two or more sinks, such as [Apache Kafka](https://docs.ververica.com/user_guide/sql_development/connectors.html#apache-kafka) for real-time users and a [Filesystem](https://docs.ververica.com/user_guide/sql_development/connectors.html#file-system) for offline users.
+Othertimes, two queries are not the same but share some extensive intermediate operations.
+
+When working with server logs, the support team would like to see the number of status codes per browser every 5 minutes to have real-time insights into web pages' status. 
+Additionally, they would like the same information on an hourly basis made available as partitioned [Apache Parquet](https://docs.ververica.com/user_guide/sql_development/connectors.html#apache-parquet) files so they can perform historical analysis. 
+
+We could quickly write two Flink SQL queries to solve both these requirements, but that would not be efficient. 
+These queries have a lot of duplicated work - reading the source logs topic and cleansing the data. 
+
+Ververica Platform Stream edition offers a feature called `STATEMENT SET`s that allow for multiplexing `INSERT INTO` statements into a single query holistically optimized by Apache Flink and deployed as a single application. 
```

Comment:
```suggestion
Ververica Platform includes a feature called `STATEMENT SET`s, that allows for multiplexing `INSERT INTO` statements into a single query holistically optimized by Apache Flink and deployed as a single application. 
```

Reply:
Think there's no need to mention the edition here, just mention it's available on VVP.